Automattic/jetpack
# Blog onboarding: Add design-first launchpad definition

Fixes https://github.com/Automattic/dotcom-forge/issues/2140

## Proposed changes:

* This PR adds the `design-first` Launchpad definition to the Launchpad API.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?


No.

## Testing instructions:






1. Run `bin/jetpack-downloader test jetpack-mu-wpcom-plugin add/design-first-launchpad` in your WPCOM Sandbox to build and sync this branch to your sandbox. 
2. Sandbox the API
3. Load this Calypso PR locally to test the design-first flow. https://github.com/Automattic/wp-calypso/pull/77222
4. Start a new onboarding site by logging out, navigating to /setup/design-first, and signing up for a new account
5. Go to `https://developer.wordpress.com/docs/api/console/`, select Rest API and wpcom/v2, and input the following: `/sites/YOURSITESLUG/launchpad/checklist?checklist_slug=design-first`. Confirm you get back an appropriate checklist that matches the status of checklist items for the `design-first` flow
